src/services/claims.js

```diff
@@ -0,0 +1,42 @@
+const getRolesFromClaims = claims => {
+  const keys = Object.keys(claims);
+  const roleClaims = [];
+
+  if (keys.includes('officer')) {
+    roleClaims.push('officer');
+  }
+
+  if (keys.includes('member')) {
+    roleClaims.push('member');
+  }
+
+  if (keys.includes('inductee')) {
+    roleClaims.push('inductee');
+  }
+
+  return roleClaims;
+};
+
+const isAnOfficer = userContext => {
```

Comment:
why not isOfficer? jw

Reply:
Since InducteePoints already has an isOfficer field for its state, I can't do something like `this.setState({ isOfficer: isOfficer(userContext) })` apparently : (

Reply:
ah that's fine then - I was thinking it might be nicer to just do change that stateField cuz isAnOfficer sounds a little weird

Reply:
Yeah I did that, changed the stateField from isOfficer to isAnOfficer and the function names are back to being isOfficer, isMember and isInductee.